content/automate/ManageIQ/System/Notification/Email.class/__methods__/miq_provision_complete.rb

```diff
@@ -0,0 +1,97 @@
+#
+# Description: This method sends an e-mail when the following event is raised:
+# Events: vm_provisioned
+#
+# Model Notes:
+# 1. to_email_address - used to specify an email address in the case where the
+#    vm's owner does not have an  email address. To specify more than one email
+#    address separate email address with commas. (I.e. admin@example.com,user@example.com)
+# 2. from_email_address - used to specify an email address in the event the
+#    requester replies to the email
+# 3. signature - used to stamp the email with a custom signature
+#
+
+module ManageIQ
+  module Automate
+    module System
+      module Notification
+        module Email
+          class MiqProvisionComplete
+            def initialize(handle = $evm)
+              @handle = handle
+            end
+
+            def main
+              @handle.log("info", "Starting miq_provision_complete")
+              check_email
```

Comment:
@billfitzgerald0120 
This method is only setting the body so should we just change the check_email method to be called
build_body